### PR TITLE
Update upgrade instructions for 6.0. 

### LINKED ISSUE #183: Update Upgrade instructions for Citus 6
### Upgrading Citus

https://docs.citusdata.com/en/v5.2/admin_guide/upgrading_citus.html

We need to update Major upgrade instructions from Citus 5 to Citus 6. @mtuncer Could you add a note to this issue when you have tested upgrade from Citus 5 to Citus 6 on steps needed?
### Upgrading Postgres

Now that we are fully extensified, we can consider having a separate section for just Upgrading Postgres, and point them towards existing pg_upgrade documentation.

### PR DESCRIPTION
**Pull request by sumedhpathak**
Restructure section to upgrade Citus versions and PostgreSQL versions.
Add a contact-us for now on PostgreSQL version upgrade.

Fixes #183

### CODE REVIEW

File: admin_guide/upgrading_citus.rst [hunk 1]
```diff
@@ -197,15 +41,33 @@ Step 1. Update Citus Package
...
 the extension on all database instances in your cluster.
+
+
+
+.. _upgrading_citus_minor:
+
+Upgrading PostgreSQL version from 9.5 to 9.6
+############################################
+
+Citus v6.0 is compatible with PostgreSQL 9.5.x and 9.6.x. If you are running
```
Comment (samay-sharma) [inline admin_guide/upgrading_citus.rst]: (Minor) If you are running Citus on PostgreSQL version 9.5 and wish ...

File: admin_guide/upgrading_citus.rst [hunk 2]
```diff
@@ -5,167 +5,11 @@ $$$$$$$$$$$$$$$
 
 .. _upgrading_citus_major:
```
Comment (samay-sharma) [inline admin_guide/upgrading_citus.rst]: Change tags to match section header

File: admin_guide/upgrading_citus.rst [hunk 3]
```diff
@@ -197,15 +41,33 @@ Step 1. Update Citus Package
...
   psql -c "\dx"
-  # you should see a newer Citus 5.x version in the list
+  # you should see a newer Citus 6.0 version in the list
 
 That's all it takes! No further steps are necessary after updating
 the extension on all database instances in your cluster.
+
+
+
+.. _upgrading_citus_minor:
```
Comment (samay-sharma) [inline admin_guide/upgrading_citus.rst]: Change tags to match section header

**Comment by sumedhpathak** [comment]
Addressed review feedback. Merging.

**Pull request merged**